src/Pos/Txp/Toil/Trans.hs

```diff
@@ -101,7 +102,7 @@ instance MonadBalancesRead m => MonadBalances (TxpT m) where
 instance Monad m => MonadTxPool (TxpT m) where
     hasTx id = TxpT $ use $ txmMemPool . mpLocalTxs . to (HM.member id)
 
-#ifdef DWITH_EXPLORER
+#ifdef WITH_EXPLORER
     putTxWithUndo id tx undo extra = TxpT $ do
```

Comment:
why `MonadTxExtra` exists and this method takes extra at the same time?
seems after creating of `MonadTxExtra`, extra parameter isn't needed more.